integration

```diff
@@ -133,6 +133,7 @@
   "bremor/bonaire_myclimate",
   "bremor/bureau_of_meteorology",
   "bremor/public_transport_victoria",
+  "brenard/ccei-tild-custom-component",
```

Comment:
<https://github.com/brenard/ccei-tild-custom-component/blob/2024.1.7/hacs.json#L3>
HACS is currently at 1.34.0, so it will take a (good) while before that will be a valid minimum version.

Reply:
:smile: I fixed it  https://github.com/brenard/ccei-tild-custom-component/commit/10e86530b41a897b4e94efda2573f4208cd28d99

Reply:
Its not part of a release.

Reply:
> Its not part of a release.

I added it in a new 2024.9.1 release